src/puppetlabs/puppetdb/mq_listener.clj

```diff
@@ -353,19 +354,22 @@
 
 (defservice message-listener-service
   MessageListenerService
-  [[:PuppetDBServer shared-globals]]
+  [[:DefaultedConfig get-config]
+   [:PuppetDBServer]]
```

Comment:
I thought it was necessary, but perhaps I was wrong.  At least removing it now doesn't hurt the unit tests.